demo/ui/utils/config.py

```diff
@@ -0,0 +1,42 @@
+import tomllib
+from pathlib import Path
+
+from utils import constants
+
+
+def get_project_root() -> Path:
+    """Get the project root directory"""
+    return Path(__file__).resolve().parent.parent
+
+PROJECT_ROOT = get_project_root()
+
+class Config:
+
+    address = []
+
+    def __init__(self):
+        self._load_initial_config()
+
+    @staticmethod
+    def _get_config_path() -> Path:
+        root = PROJECT_ROOT
+        config_path = root / "config" / "config.toml"
+        if config_path.exists():
+            return config_path
+        raise FileNotFoundError("No configuration file found in config directory")
+
+    def _load_initial_config(self):
+        config_path = Config._get_config_path()
+        with config_path.open("rb") as f:
+            raw_config = tomllib.load(f)
```

Comment:
Let's put the entire **with ...** inside try except block as tomllib.load() can return different types of errors and we should catch those errors return a user-friendly error message back to client to avoid leakage of any critical info